CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on you Machine:
+
+1. Python3 and pip3
+2. Docker or Podman
+3. Ansible
+   This can be installed by running `pip3 install ansible`
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<root directory>` with the actual path where you want to clone the repositories and set up your environment. 
+
+1. **Create Directories**:
+
+   Create the following directories in your home directory:
+
+   ```shell
+   mkdir -p <root directory>/ansible_collections/cloud/azure_ops
+   mkdir -p <root directory>/ansible_collections/azure/azcollection
+   ```
+
+2. **Clone This Repository**:
+   ```shell
+   git clone https://github.com/redhat-cop/cloud.azure_ops.git <root directory>/ansible_collections/cloud/azure_ops
+   ```
+
+3. ***Clone Azure Collection Repository***:
+   ```shell
+   git clone https://github.com/ansible-collections/azure.git <root directory>/ansible_collections/azure/azcollection
+   ```
+
+4. ***Prepare Azure Configuration File***:
+   Prepare the Azure configuration file at `<root directory>/ansible_collections/cloud/azure_ops/tests/integration/cloud-config-azure.ini`. A template of this file is available in [the Ansible repository](https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/config/cloud-config-azure.ini.template). Populate the file with your appropriate credentials and resource group information.
+
+## Running tests
+
+To run the tests, change directory into azure-ops directory:
+```shell
+   cd <root directory>/ansible_collections/cloud/azure_ops
+```
+
+### Integration tests
```

Comment:
Maybe add some instructions on how to run the specific test?

Reply:
ansible-test integration <name of the specific test> --docker

Reply:
`ansible-test integration <name of the specific test> --docker`

Reply:
Added, thanks